codegen/src/generator/rest_json.rs

```diff
@@ -13,7 +13,7 @@ impl GenerateProtocol for RestJsonGenerator {
             let output_type = operation.output_shape_or("()");
 
             // Retrieve the `Shape` for the input for this operation.
-            let input_shape = service.shapes.get(input_type).unwrap();
+            let input_shape = &service.shapes[input_type];
```

Comment:
Well you can use indexes on maps (not sure if that's a new thing), and if it fails it'll panic anyways, so this method is cleaner, IMO :)